### ISSUE TITLE
Implement ex1 training

### ISSUE THREAD

**Issue by DrJonnyT** [AI-Fundamentals/DiffPrivNPUserModeling#24]
Note that the model dimensions are slightly different due to the data dimensions

**Comment by DrJonnyT** [comment]
Can you just have a setting for x dim and y dim, and everything else is literally exactly the same?

**Comment by DrJonnyT** [comment]
I have done this in the script (dimx and dimy) but need to go through them all and standardize e.g. it's currently called model_ex2 and at the end it says "finished experiment2 training script"

**Comment by DrJonnyT** [comment]
Seems like ex1 doesn't train past about accurayc of 0.4
I've tried changing privacy params, no change
I've tried changing dim_lv- default is 0, default in neuralprocesses is 128. But when set to 128 it's unstable, after some number of epochs the loss goes really high then goes to nan

**Comment by DrJonnyT** [comment]
I tried saving the exact same numbers from Julia into hdf and training in python (pre-batched). Same issue, doesn't get accuracy better than about 0.4. Therefore the problem must be with the model not the data?
